vars/Amazon-2.yml

```diff
@@ -6,6 +6,7 @@ supervisor_epel_package: >
   https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{supervisor_epel_release}}.noarch.rpm
 
 supervisor_packages:
+  - python2-pip
```

Comment:
This can't be right, if supervisor is installed from the system package pip isn't needed, or if needed it would be a package dependency already.

Also, the role does work, I'm using it for the RTB role.

Reply:
Um... on the other hand the rtb role already has `python2-pip` as dependency, so maybe that's why it works.